### ISSUE TITLE
Can't run 3.23.0's javac replacement

### ISSUE THREAD

**Issue by amalloy** [typetools/checker-framework#5216]
After unzipping the release (https://github.com/typetools/checker-framework/releases/download/checker-framework-3.23.0/checker-framework-3.23.0.zip), I find a javac replacement as expected, in `checker/bin/javac`. But when I run that binary, it crashes regardless of inputs:

```
$ checker-framework-3.23.0/checker/bin/javac cf/CFRepro.java 
Exception in thread "main" java.lang.RuntimeException: The following files could not be located: /home/amalloy/src/java/checker-framework-3.23.0/checker/dist/checker-qual.jar, /home/amalloy/src/java/checker-framework-3.23.0/checker/dist/checker-util.jar
        at org.checkerframework.framework.util.CheckerMain.assertFilesExist(CheckerMain.java:759)
        at org.checkerframework.framework.util.CheckerMain.assertValidState(CheckerMain.java:161)
        at org.checkerframework.framework.util.CheckerMain.<init>(CheckerMain.java:153)
        at org.checkerframework.framework.util.CheckerMain.main(CheckerMain.java:61)
```

Indeed, when I look in the `checker/dist` directory, I do not see these jars: 

```
$ ls checker-framework-3.23.0/checker/dist/
checker.jar  checker-javadoc.jar  checker-source.jar
```

**Comment by cushon** [comment]
I just noticed this as well, it seems like a regression since 3.22.0:

```
$ ./checker-framework-3.22.0/checker/bin/javac -processor Nullness T.java
....
OK

$ ./checker-framework-3.23.0/checker/bin/javac -processor Nullness T.java
Exception in thread "main" java.lang.RuntimeException: The following files could not be located: /tmp/tmp.vxHJGsXh3h/checker-framework-3.23.0/checker/dist/checker-qual.jar, /tmp/tmp.vxHJGsXh3h/checker-framework-3.23.0/checker/dist/checker-util.jar
        at org.checkerframework.framework.util.CheckerMain.assertFilesExist(CheckerMain.java:759)
        at org.checkerframework.framework.util.CheckerMain.assertValidState(CheckerMain.java:161)
        at org.checkerframework.framework.util.CheckerMain.<init>(CheckerMain.java:153)
        at org.checkerframework.framework.util.CheckerMain.main(CheckerMain.java:61)
```

**Comment by mernst** [comment]
Thanks for the bug report, and our apologies for the problem.  It is corrected in Checker Framework 3.24.0, which was released yesterday.
